### PR TITLE
Add cron resource

### PR DESCRIPTION
**Pull request by rmoe**
This adds the ability to manage cron resources including importing them.

### CODE REVIEW

File: travis/resource_cron.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/shuheiktgw/go-travis"
+)
+
+func resourceCron() *schema.Resource {
+	return &schema.Resource{
+		Description: "The `travis_cron_job` resource creates a cron job for a branch.",
```
Comment (bgpat) [inline travis/resource_cron.go:17]:
```suggestion
		Description: "The `travis_cron` resource creates a cron job for a branch.",
```

File: travis/resource_cron.go [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+			},
+			"branch": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The branch to which this cron job belongs.",
+				ForceNew:    true,
+			},
+			"interval": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
```
Comment (bgpat) [inline travis/resource_cron.go:46]: The serevr returns an error without interval.

> 422 error Invalid value for interval. Interval must be "daily", "weekly" or "monthly"!

```suggestion
				Required:     true,
```

Comment (rmoe) [reply travis/resource_cron.go:46]: Nice catch.

**Comment by rmoe** [comment]
@bgpat I think I've addressed your feedback, could you please take another look at this?

**Review by bgpat — APPROVED** [review]
LGTM

**Pull request merged**